README.md

```diff
@@ -90,6 +95,10 @@ docker exec -it <container-name> bash
 ansible-playbook -i /root/code/casl-ansible/inventory/sample.casl.example.com.d/inventory /root/code/casl-ansible/playbooks/openshift/end-to-end.yml -e openstack_ssh_public_key=<your_ssh_key_name>
 ```
 
+```
+ansible-playbook -i /root/code/casl-ansible/inventory/ample.aws.example.com.d/inventory /root/code/casl-ansible/playbooks/openshift/end-to-end.yml --private-key==<your_ssh_private_key_file>
```

Comment:
Not sure what we want two commands listed here as that will be confusing. Maybe you can make the existing one a bit more generic - i.e.: something like `-i <inventory>` with a comment below that shows something like : 
```
where `<inventory>` is the path to your inventory directory as mentioned above.

Reply:
Yes, I agree. Let's make it more generic and also remove the --private-key as suggested below

Reply:
@oybed what about 'openstack_ssh_public_key'? Should we remove it as well and add that to the README's instructions?

Reply:
@makentenza yes, that would be good. Although, we'd also need to supply the public key for AWS (and most likely other cloud environments as well), so maybe we just make this parameter more generic, i.e.: `public_ssh_key` and hence it can *always* be supplied. What do you think?

Reply:
@oybed in this case we need to supply the private key. I think the public key is something that is only required for the 'openstack-stack' role in openshift-ansible-contrib if I'm not wrong. For other IaaS providers, including OSP, we will need to specify the private key.